docs/source/search-runs.rst

```diff
@@ -93,139 +93,183 @@ Then paste the following code to a local pythnon file, e.g., ``~/create_testing_
+   :alt: search query
+   :width: 90%
+   :align: center
 
-- An MLflow field on the left side, which can be some metric, param, tag, dataset or MLflow run metadata, e.g.,
-  ``metric.loss`` or ``param.batch_size``.
-- A comparator in the middle, we support 3 types of comparators:
+You'll notice that we now are displaying 2 runs instead of 10. Pretty easy, right?
 
-  - Numeric comparators: ``=``, ``!=``, ``>``, ``>=``, ``<``, and ``<=``.
-  - String comparators: ``=``, ``!=``, ``LIKE`` and ``ILIKE``. String comparators are case sensitive except ``ILIKE``
-    which is case insensitive.
-  - Set comparators: ``IN``, which only supports searching by dataset and run metadata, and is case sensitive.
+Now let's go over the search query syntax in more detail.
 
-- A reference value on the right side.
+Search Query Syntax Deep Dive
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. note::
-  There is a caveat that using ``IN`` requires wrapping string with single quotes, e.g.,
-  ``params."learning rate" IN ('0.001', '0.01')``. ``params."learning_rate" IN ("0.001", "0.01")`` will throw an error.
+As noted above, MLflow search syntax is similar to SQL with a few notable exceptions.
 
-Handling Special Characters in MLflow Field
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* The SQL ``OR`` keyword is not supported.
+* For fields that contain special characters or start with numbers, you need to wrap them in double quotes.
 
-It's common that your MLflow field contains special characters such as hyphen, space, period, and so on. In those
-cases, you will need to wrap it by double quotes. The same rule applies when your field starts with numbers.
-For example if you ever logged something by ``mlflow.log_metric("cross-entropy-loss", 0.15)``, then your query will be
-similar to:
+  * Good: ``metrics."cross-entropy-loss" < 0.5``
+  * Bad:  ``metrics.cross-entropy-loss < 0.5``
+  * Good: ``params."1st_iteration_timestamp" = "2022-01-01"``
+  * Bad:  ``params.1st_iteration_timestamp = "2022-01-01"``
 
-.. code-block:: sql
+* For the SQL ``IN`` keyword, you must surround the values of your list with single quotes.
+
+  * Good: ``params."learning rate" IN ('0.001', '0.01')``
+  * Bad:  ``params."learning rate" IN ("0.001", "0.01")``
+
+* For the SQL ``IN`` keyword, you can only search the following fields:
+
+  * ``datasets.{any_attribute}``
+  * ``attributes.run_id``
+
+* Non-None conditions for numeric fields are not supported e.g. ``metrics.accuracy != "None"`` will fail.
+
+Other than the that, the syntax should be intuitive to anyone who has used SQL. To assemble
+a single search condition, you must assemble an inequality using the following components...
+
+1. **An MLflow field**: a metric, param, tag, dataset or run metadata.
+2. **A comparator**: an inequality operator. 
+
+  * For numerics, MLflow supports ``=``, ``!=``, ``>``, ``>=``, ``<``, and ``<=``.
+  * For strings, MLflow supports ``=``, ``!=``, ``LIKE`` (case-sensitive) and ``ILIKE`` (case-insensitive). 
```

Comment:
Let's provide some examples of common query terms (particularly LIKE and ILIKE) and some range search conditions (metric_A >= 0.7654, etc.)